specs/change_password/02_user_interface.rst

```diff
@@ -0,0 +1,136 @@
+
+.. _74: https://github.com/gnosis/safe/issues/74
+
+.. _Main:
+
+
+#. `Main`_
+#. `User Interface`_
+#. `Technical Details`_
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Current password
+------------------
+
+User has to enter the current password.
+
+.. raw:: html
+
+   <img src="screens/android/current_password.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/current_password.png" width="320px"/>
+
+* Zeplin link Android: https://zpl.io/bJK86GW
+* Zeplin link iOS: https://zpl.io/aNP44ZQ
+
+Current password - Invalid
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+User has not entered the current password correctly.
+
+.. raw:: html
+
+   <img src="screens/android/current_password_invalid.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/current_password_invalid.png" width="320px"/>
+
+* Zeplin link Android: https://zpl.io/aBEv3KK
+* Zeplin link iOS: https://zpl.io/b61lQOd
+
+
+New password
+------------------
+
+User has to enter new password and confirm.
+
+.. raw:: html
+
+   <img src="screens/android/new_password.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/new_password.png" width="320px"/>
+
+* Zeplin link Android: https://zpl.io/aR1NNXK
+* Zeplin link iOS: https://zpl.io/bJ7NOeW
+
+New password - Invalid
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+User has entered a new password that does not meet the requirements.
+
+.. raw:: html
+
+   <img src="screens/android/new_password_invalid.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/new_password_invalid.png" width="320px"/>
+
+* Zeplin link Android: https://zpl.io/aR1NNXK
+* Zeplin link iOS: https://zpl.io/bJ7NOeW
+
+New password - Success
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+User has entered a new password that does meet the requirements.
+
+.. raw:: html
+
+   <img src="screens/android/new_password_success.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/new_password_success.png" width="320px"/>
+
+* Zeplin link Android: https://zpl.io/b614l1m
+* Zeplin link iOS: https://zpl.io/25OggEW
+
+New password - Confirm invalid
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+User has confirmed a password that does match with the new one.
+
+.. raw:: html
+
+   <img src="screens/android/new_password_confirm_invalid.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/new_password_confirm_invalid.png" width="320px"/>
+
+* Zeplin link Android: https://zpl.io/aXP3GEP
+* Zeplin link iOS: https://zpl.io/2EBwy7x
+
+New password - Confirm success
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+User has successfully entered and confirmed a new password.
+
+.. raw:: html
+
+   <img src="screens/android/new_password_confirm_success.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/new_password_confirm_success.png" width="320px"/>
+
+* Zeplin link Android: https://zpl.io/2jQlk3A
```

Comment:
Android screen does not match the screenshot

Reply:
lol Michel changed the screen comepltely ..

Reply:
Oh I see... 🤔